lint-requirements.txt

```diff
@@ -1,5 +1,5 @@
 bandit==1.7.5
 black==23.3.0
-codespell==2.2.4
-flake8==6.0.0
-isort==5.12.0
+codespell==2.2.5
+flake8==5.0.4
+isort==5.11.4
```

Comment:
these should remain the newest versions, not be downgraded

Reply:
oh right I saw the version in `isort` I though that would keep the latest one but no I should keep the latest on those version.
That's why I put it into review :laughing: